components/gif_picker/components/SearchGrid/SearchGrid.tsx

```diff
@@ -149,12 +147,13 @@ export class SearchGrid extends PureComponent {
         } = this.props;
 
         const {containerWidth} = this.state;
-        const {moreRemaining, items = [], isEmpty = items.length === 0, isFetching} = resultsByTerm[keyword] ? resultsByTerm[keyword] : {};
+        const {moreRemaining, items = [], isFetching} = resultsByTerm[keyword] ? resultsByTerm[keyword] : {} as Partial<GifsResult>;
+        const isEmpty = items.length === 0;
 
         /**
          * Columns 'left' values
          */
-        const columnWidth = parseInt(containerWidth / this.numberOfColumns, 10);
```

Comment:
why changed from parseInt to floor?

Reply:
Both "containerWidth" and "this.numberOfColumns" are numbers, I'm assuming that "parseInt" was used to round the number to an int type instead of using the resulting float from the divide operation.
Floor keeps the rounding functionality same as parseInt, an alternative would be to convert it to string and then parse it as so:
```
const columnWidth = parseInt(`${containerWidth / this.numberOfColumns}`, 10);
```
Let me know what you would like me to do here.

Reply:
yes i checked both values are always number, so it safe to use floor